### ISSUE TITLE
master branch

### ISSUE THREAD

**Issue by saraedum** [sagemath/sage-archive-2023-02-01#9]
Could we rename the "working" branch to be called "master" again? The dev scripts assume that the main branch is called master and I believe that this is what most people expect.
I have to admit that I do not understand what the master branch is currently...

**Comment by ohanar** [comment]
No -- there are commits on "working" that will be dropped, as they are hacks to get around http://trac.sagemath.org/14781. I'm planning on reviewing this after I finish teaching for the day, and if everything checks out, I'll preemptively merge it into "master" -- then the "working" branch can be deleted, and people can develop off of "build_system".

The "master" branch includes only changes that have been exported from the mercurial repos. This is done for two reasons:
1. New releases of sage always simple merges into the "master" branch
2. It is possible to review the changes to the build system (at http://trac.sagemath.org/14480) using the new workflow. Once this is positively reviewed, I'll close this ticket. After this, I will open a new ticket for any release of sage that is not a simple merge.

**Comment by saraedum** [comment]
I'm not sure I understand. Should I change the sagedev scripts to treat `working` as the default branch? At least for the time being?

**Comment by saraedum** [comment]
sorry, I meant to say `build_system`

**Comment by ohanar** [comment]
The main point is that anything in `master` should have been reviewed, however the changes that I've made to the build system have yet to be reviewed. As a temporary workaround, we could change the dev scripts to use `build_system` instead.

Also, if you are planning on working on the dev scripts, I would very much suggest looking at the branch I've attached to [#14482](http://trac.sagemath.org/14482), since I've more or less finished the git and trac backends, including doctests.
